src/shared/components/mutationTable/MutationTable.tsx

```diff
@@ -0,0 +1,132 @@
+            },
+            proteinChange: {
+                name: "Protein Change",
+                formatter: ProteinChangeColumnFormatter,
+                sortable: ProteinChangeColumnFormatter.sortFunction,
+                filterable: true,
+                visible: true
+            },
+            startPos: {
+                name: "Start Pos",
+                dataField: "startPos",
+                sortable: true,
+                filterable: false,
+                visible: true
+            },
+            endPos: {
+                name: "End Pos",
+                dataField: "endPos",
+                sortable: true,
+                filterable: false,
+                visible: true
+            },
+            referenceAllele: {
+                name: "Ref",
+                dataField: "referenceAllele",
+                sortable: false,
+                filterable: false,
+                visible: true
+            },
+            variantAllele: {
+                name: "Var",
+                dataField: "variantAllele",
+                sortable: false,
+                filterable: false,
+                visible: true
+            },
+            mutationStatus: {
+                name: "MS",
+                dataField: "mutationStatus",
+                sortable: true,
+                filterable: true,
+                visible: true
+            },
+            validationStatus: {
+                name: "VS",
+                dataField: "validationStatus",
+                sortable: true,
+                filterable: true,
+                visible: true
+            },
+            center: {
+                name: "center",
+                sortable: true,
+                filterable: true,
+                visible: true
+            },
+            mutationAssessor: {
+                name: "Mutation Assessor",
+                formatter: MutationAssessorColumnFormatter,
+                sortable: MutationAssessorColumnFormatter.sortFunction,
+                filterable: true,
+                visible: true
+            }
+        };
+    };
+
+    constructor(props:IEnhancedReactTableProps)
+    {
+        super(props);
+        this.state = {};
+    }
+
+    public render()
+    {
+        const {reactTableProps, columns, rawData} = this.mergeProps(this.props);
+
+        return(
+            <EnhancedReactTable
+                reactTableProps={reactTableProps}
+                columns={columns}
+                rawData={rawData}
+            />
+        );
+    }
+
+    private mergeProps(props:IEnhancedReactTableProps):IEnhancedReactTableProps
+    {
+        const defaultProps:IEnhancedReactTableProps = {
+            rawData: [],
+            columns: MutationTable.defaultColumns,
+            reactTableProps: {
+                className: "table",
+                id: "table"
+            }
+        };
+
+        // merge provided props with the default props
+        return lodash.merge(defaultProps, props);
```

Comment:
Just realized the defaultColumns is a getter - which means the in-place modification would not matter in this case.